### ISSUE TITLE
Is it the correct way to annotate Manhattan plot with significance level?

### ISSUE THREAD

**Issue by garyzhubc** [privefl/bigsnpr#466]
Is it the correct way to annotate Manhattan plot with significance level?`snp_manhattan(...) + geom_hline(yintercept = -log10(5e-8), linetype = 2, color = "red")`

**Comment by privefl** [comment]
Yes, you can do that if you want to visually add the threshold of genome-wide significance.

I'll add that to the examples.

**Comment by garyzhubc** [comment]
Why not add that as an option (also the title)? Also, the index taking seems quire redundant because you can just get it from the bigsnpr object.

**Comment by privefl** [comment]
Because this is a `ggplot` object that you get, and you can add all this with the {ggplot2} functions, so why bother adding many parameters in my functions to do the same thing?
